src/priorityQueue.js

```diff
@@ -0,0 +1,72 @@
+export class Element {
+  constructor(value, priorityInt) {
+    this.data = value
+    this.priority = priorityInt
+  }
+}
+
+export default class PriorityQueue {
+  constructor() {
+    this.elements = []
+    this.size = 0
+  }
+
+  enqueue(value, priorityInt) {
+    const element = new Element(value, priorityInt)
+    this.elements.push(element)
+    this.size += 1
+  }
+
+  front = () => {
+    if(this.size === 0) return null
+
+    let priority = this.elements[0].priority
+    let element = this.elements[0]
+
+    for(let j = 1; j < this.size; j++){
+      if(this.elements[j].priority < priority){
+        priority = this.elements[j].priority
+        element = this.elements[j]
+      }
+    }
+    return element.data
+  }
+
+  back = () => {
+    if(this.size === 0) return null
+
+    let priority = this.elements[0].priority
+    let element = this.elements[0]
+
+    for(let j = 1; j < this.size; j++){
+      if(this.elements[j].priority > priority){
+        priority = this.elements[j].priority
+        element = this.elements[j]
+      }
+    }
+    return element.data
+  }
+
+  dequeue() {
```

Comment:
This implementation is confusing to me - I'm not sure how it's supposed to work after a few reads...  There may be an opportunity here to change some identifiers in order to more clearly specify the behavior of this function?  As well, when I see the same prefix repeated over and over within a given context, I tend to omit it entirely - it's clear that `data`, `element`, and `index` are being used for dequeueing given that this is being executed in the `dequeue` function itself, so we can omit the prefix for more concise identifier names.